### PR TITLE
webhooksonlists

### PR DESCRIPTION
**Pull request by learafa**
added documentations for support of webhooks on lists

### CODE REVIEW

**Comment by Lauragra** [comment]
@learafa , do you plan to make the same update to the v1.0 files? Also, I see that your Permissions tables are out of alignment with the standard structure. In the future, this might cause validation errors as we will be enforcing a standard structure across the docs.

**Comment by Lauragra** [comment]
Please include an update to the changelog if this is an API change.

**Comment by Lauragra** [comment]
@learafa , @piotrci , what is the status of this PR? Do we need to get it merged in?

File: concepts/changelog.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 | :-------------- | :---------- | :--------------------------------------- |
 | Addition        | Beta        | Added the [followSite](/graph/api/follow-site?view=graph-rest-beta) navigation property to the [user](/graph/api/resources/user?view=graph-rest-beta) entity set |
 | Addition        | Beta        | Added the [unfollowSite](/graph/api/unfollow-site?view=graph-rest-beta) navigation property to the [user](/graph/api/resources/user?view=graph-rest-beta) entity set |
+| Addition        | v1.0        | Added the **subscriptions** navigation property to the [list](/graph/api/resources/list?view=graph-rest-beta) entity |
```
Comment (Lauragra) [inline concepts/changelog.md]: @learafa , please update the list resource topic to include the new navigation property. Please ensure that all the navigation properties on the list resource are reflected in the resource topic.

File: concepts/changelog.md [hunk 2]
```diff
@@ -849,6 +849,12 @@ Debut of the places API which provides rich details for locations in apps.
 | :-------------- | :------------ | :--------------------------------------- |
 | Addition | beta | Added support for the Mail.ReadBasic.All application permission in mailFolder API: [list mailfolders](/graph/api/user-list-mailfolders?view=graph-rest-beta), [get a mailfolder](/graph/api/mailfolder-get?view=graph-rest-beta), [list child folders](/graph/api/mailfolder-list-childfolders?view=graph-rest-beta) and [list messages in a mail folder](/graph/api/mailfolder-list-childfolders?view=graph-rest-beta). Also added Mail.ReadBasic.All support in [delta query for message](/graph/api/message-delta?view=graph-rest-beta) and [delta query for mailFolder](/graph/api/mailfolder-delta?view=graph-rest-beta).|
 
+### Sites (SharePoint)
+
+| **Change type** | **Version** | **Description**                          |
+| :-------------- | :---------- | :--------------------------------------- |
+| Addition        | v1.0        | Added the **subscriptions** navigation property to the [list](/graph/api/resources/list?view=graph-rest-beta) entity. |
```
Comment (baywet) [inline concepts/changelog.md]:
```suggestion
| Addition        | beta       | Added the **subscriptions** navigation property to the [list](/graph/api/resources/list?view=graph-rest-beta) entity. |
```

Comment (baywet) [reply concepts/changelog.md]: @learafa does this make sense? as the changes have only been made in beta and not in v1.0?

**Comment by baywet** [comment]
@learafa can you add your requests paths to api-reference/beta/resources/webhooks.md as well please?

File: concepts/changelog.md (hunk 2 above)
Comment (baywet) [reply concepts/changelog.md]: confirmed on teams, only in beta at the moment

**Pull request merged**